datasets/ee/riigikogu/crawler.py

```diff
@@ -0,0 +1,102 @@
+from zavod import Context, helpers as h
+from zavod.logic.pep import categorise
+
+def get_members_url(context: Context) -> list:
+    """
+    Fetches the URLs for the pages of each member from the main page.
+
+    :param context: The context object.
+
+    :return: The URLs for the pages.
+    """
+
+    try:
+        main_website = context.fetch_html(context.data_url)
+
+    except:
+        context.log.error("Couldn't fetch main website")
+        return None
+
```

Comment:
no need for you to handle exceptions from http requests generally - an error in fetch_html will raise an exception handled by the crawler runner, resulting in an error log and alert on the website at https://www.opensanctions.org/issues/